code/modules/projectiles/guns/special/meat_hook.dm

```diff
@@ -77,3 +78,26 @@
 /obj/projectile/hook/bounty
 	damage = 0
 	stamina = 40
+
+/// non-damaging version for contractor MODsuits
+/obj/item/gun/magic/hook/contractor
+	name = "SCORPION hook"
+	desc = "A hardlight hook used to non-lethally pull targets much closer to the user."
+	icon_state = "contractor_hook"
+	inhand_icon_state = "" //nah
+	ammo_type = /obj/item/ammo_casing/magic/hook/contractor
+
+/obj/item/ammo_casing/magic/hook/contractor
+	projectile_type = /obj/projectile/hook/contractor
+
+/obj/projectile/hook/contractor
+	icon_state = "contractor_hook"
+	damage = 0
+	stamina = 25
+	chain_iconstate = "contractor_chain"
```

Comment:
This one's optional, but could you maybe change the `hitsound` for this? It doesn't feel like it should be making a 'splat' sound when it hits something.

Reply:
WIll do when I find something better (open to suggestions)